server/routes.go

```diff
@@ -192,6 +237,7 @@ func Serve(ln net.Listener) error {
 	r.POST("/api/generate", generate)
 	r.POST("/api/create", create)
 	r.POST("/api/push", push)
+	r.GET("/api/tags", list)
```

Comment:
do we want to use `/api/models`?

Reply:
I put it as tags since you're not getting the entire model. We may want to rethink the REST API though since it's not particularly RESTful right now.